SongLab/Components/Home/HomeView.swift

```diff
@@ -31,19 +31,17 @@ struct HomeView: View {
             RecordingsListView(
                 audioManager: viewModel.audioManager,
                 recordingManager: viewModel.recordingManager,
-                selectedSession: $viewModel.selectedSession
+                selectedSession: $viewModel.selectedSession,
+                appTheme: $viewModel.appTheme
```

Comment:
I think you should go ahead and implement user selection of the theme as part of this PR.